bicep/README.md

```diff
@@ -0,0 +1,20 @@
+The files in this folder are intended as a getting started template for developing and deploying bicep modules on Azure DevOps.
+
+# Local development
+
+For local development, the file [local.ps1](./deploy/local.ps1) can be used to run the deployment locally.
+
+The --what-if command can be uncommented to show changes that would be deployed.
+
+## Pre-Requisites
+
+- Login to azure using az cli
+- Fill out the development subscription id in [local.ps1](./deploy/local.ps1)
+
+# Azure DevOps
+
+The [azure-pipeline.yml](azure-pipeline.yml) file contains the steps necessary to build and deploy the bicep files. It references the two templates [build.yml](./deploy/templates/build.yml) and [deploy.env.yml](./deploy/templates/deploy.env.yml)
+
```

Comment:
We could add a small section "further resources" or something pointing to the official bicep quickstart templates: https://github.com/Azure/azure-quickstart-templates/tree/master/quickstarts . Realistically they will be more elaborate than what we will be able to maintain ourselves, so if people can't find what they are looking for in here, they may have more luck there. Anyway, just a thought...